rust/core-lib/src/plugins/rpc.rs

```diff
@@ -181,6 +181,9 @@ pub enum PluginNotification {
     UpdateSpans { start: usize, len: usize, spans: Vec<ScopeSpan>, rev: u64 },
     Edit { edit: PluginEdit },
     Alert { msg: String },
+    AddStatusItem { key: String, value: String, alignment: String },
```

Comment:
There's an argument that there should be a new `Alignment` type used here, but I don't think that needs to be blocking.